bot/utils/database-migrations.js

```diff
@@ -0,0 +1,88 @@
+const fs = require("fs");
+const db = require("../apis/database-crud.js");
+const Logger = require("./logger.js");
+
+/**
+ * Get a full list of available migrations, ordered from oldest to newest version.
+ * @returns {Array<string>} List of migrations, formatted like yyyy-mm-dd-name.js
+ */
+module.exports.listMigrations = () => {
+    //todo: is this ordering reliable on all OSes and all configurations?
```

Comment:
This ordering it NOT reliable on all OSes (Windows vs Linux will break). I've experienced that in Java in the past and it was a pain to figure out what was going on (it broke hitboxes on a 2d game, because our scripts were executed in the wrong order). I've checked the documentation and it does not say it's ordered, so I do not think we can rely on that.